konf/site.yaml

```diff
@@ -11,10 +11,6 @@ env:
       key: google-custom-search-key
       name: google-api
 
-  - name: FLASK_SECRET_KEY
-    secretKeyRef:
-      key: anbox-cloud-io
-      name: secret-keys
```

Comment:
I don't think this is the right solution, when removing we will fall back to the static secret in https://github.com/canonical/anbox-cloud.io/blob/f356969d83df8865d38cfae6a3bcb52b3304d87a/.env#L4 

We should rather set the key in k8s.

Reply:
Any idea why we need this key? What is it doing?

Reply:
If you follow the code, it is used as https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY

Reply:
Thanks. I'll investigate further what the issue is with the keys.

Reply:
I guess the secret is not available on the demo cluster. So it should be as easy as adding the secret there.